packages/sql-openapi/test/simple.test.js

```diff
@@ -591,3 +591,52 @@ test('deserialize JSON columns', { skip: isSQLite }, async (t) => {
     same(json[0].metadata, jsonData)
   }
 })
+
+test('expose the api with a prefix, if defined', async (t) => {
+  const { pass, teardown, same, equal } = t
+
+  const app = fastify()
+  app.register(sqlMapper, {
+    ...connInfo,
+    async onDatabaseLoad (db, sql) {
+      pass('onDatabaseLoad called')
+
+      await clear(db, sql)
+      await createBasicPages(db, sql)
+    }
+  })
+  app.register(sqlOpenAPI, { prefix: '/api' })
+  teardown(app.close.bind(app))
+
+  await app.ready()
+  {
+    const res = await app.inject({
+      method: 'POST',
+      url: '/pages',
+      body: {
+        title: 'Hello'
+      }
+    })
+    equal(res.statusCode, 404, 'POST /pages status code')
+    same(res.json(), {
+      message: 'Route POST:/pages not found',
+      error: 'Not Found',
+      statusCode: 404
+    }, 'POST /pages response')
+  }
+  {
+    const res = await app.inject({
+      method: 'POST',
+      url: '/api/pages',
+      body: {
+        title: 'Hello'
+      }
+    })
+    equal(res.statusCode, 200, 'POST /pages status code')
+    equal(res.headers.location, '/api/pages/1', 'POST /api/pages location')
+    same(res.json(), {
+      id: 1,
+      title: 'Hello'
+    }, 'POST /pages response')
+  }
```

Comment:
Can you add a test for `/documentation`? Is that prefixed or not?

Reply:
I don't think `documentation` should be prefixed, but I don't have a strong opinion on this. 
However, in this PR documentation is NOT prefixed, just the `api` are, added a test to verify that, PTAL